QUICKSTART.md

```diff
@@ -0,0 +1,32 @@
+# Quick Start Guide
+
+## Pre-requisites
+
+* [Install Virtual Box 5.1.14 or later]( https://www.virtualbox.org/wiki/Downloads )
```

Comment:
Where did these specific version requirements for VBox and Vagrant come from?  As far as I know, the minimum version required by all of our software is VBox 5.0 and Vagrant 1.8.5

Are there more strict requirements now?  I think the website says something completely different too lol